docs/vhost-user-net-testing.md

```diff
@@ -0,0 +1,126 @@
+# How to test Vhost-user net with OpenVSwitch/DPDK
+
+The purpose of this document is to illustrate how to test vhost-user-net in cloud-hypervisor with OVS/DPDK as the backend.
+
+## Framework
+
+It's a simple test to validate the communication between two virtual machine, connecting them to vhost-user ports respectively provided by `OVS/DPDK`.
+```
+             +----+----------+          +-------------+-----------+-------------+          +----------+----+  
+             |    |          |          |             |           |             |          |          |    | 
+             |    |vhost-user|----------| vhost-user  |   ovs     | vhost-user  |----------|vhost-user|    |           
+             |    |net device|          | port 1      |           | port 2      |          |net device|    |           
+             |    |          |          |             |           |             |          |          |    |           
+             |    +----------+          +-------------+-----------+-------------+          +----------+    |           
+             |               |          |                                       |          |               |           
+             |vm1            |          |                  dpdk                 |          |           vm2 |           
+             |               |          |                                       |          |               |           
+          +--+---------------------------------------------------------------------------------------------+--+        
+          |  |                                       hugepages                                             |  |        
+          |  +---------------------------------------------------------------------------------------------+  |        
+          |                                                                                                   |        
+          |                                              host                                                 |        
+          |                                                                                                   |  
+          +---------------------------------------------------------------------------------------------------+        
+```
+## Prerequisites
+
+Prior to running the test, the following steps need to be performed, [vhost-user-ovs-dpdk](https://wiki.qemu.org/Documentation/vhost-user-ovs-dpdk) is a good reference for detailing them.
+- Enable hugepages
+- Install kernel-devel
```

Comment:
I am using Ubuntu.  Looks like on Ubuntu system, there is no kernel-devel.  So we may need to let people know what system the document is for.

Reply:
Yes, right, I will add this point.